core/common/src/geometry/TextString.ts

```diff
@@ -59,6 +63,10 @@ export class TextString {
   public readonly origin: Point3d;
   /** Rotation relative to element's placement */
   public readonly rotation: YawPitchRollAngles;
+  /** Optional cached text layout information. If undefined, it will be computed at display time. Default is undefined. */
```

Comment:
Can you be more specific than "cached text layout information"?

Reply:
> Can you be more specific than "cached text layout information"?

Maybe add something like "used to support legacy proxy graphics" or something like that. Nothing should use this except the V8 connector, so it doesn't need to be that detailed.

Reply:
I've added the comments as you suggested. Also, I assume no changelog entry is needed for this? I just committed ones with empty comments.

Reply:
I agree that only a V8 Connector could reasonably set these. `glyphOrigins` is pretty straightforward, but `glyphIds` is more of an internal implementation detail.